CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+## Fleet 4.60.0 (Nov 14, 2024)
+
+### Bug fixes
```

Comment:
```suggestion
```

Typo? Looks like we have the "Bug fixes and improvements section" at the bottom.